diagnosis/README.md

```diff
@@ -18,7 +22,12 @@ These are the logs retrieved by the script:
 - Kubelet service status and journal
 - Etcd service status and journal
 - Gallery item's DVM logs
+- Cluster Snapshot
 
-Please be aware that the log collector script needs to update file `~/.ssh/config` in order to connect to the cluster's worker nodes. While the script will try to back it up and then restore it once the process is complete, it may be a good a idea to create your own copy.
+## Parameters 
+1. --api-model - User can upload the apimodel.json file to Storage account
```

Comment:
Remove numbers please
Add formatting, ex: `--api-model`
Explain which ones are optional
List required parameters first
Missing -u and -i